<ui version="4.0" >
 <class>ClipPlanes</class>
 <widget class="QWidget" name="ClipPlanes" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="clearButton" >
          <property name="text" >
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="_advancedButton" >
          <property name="text" >
           <string>Advanced</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Axis</string>
        </property>
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>9</number>
         </property>
         <property name="topMargin" >
          <number>9</number>
         </property>
         <property name="rightMargin" >
          <number>9</number>
         </property>
         <property name="bottomMargin" >
          <number>9</number>
         </property>
         <property name="horizontalSpacing" >
          <number>6</number>
         </property>
         <property name="verticalSpacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <property name="horizontalSpacing" >
            <number>6</number>
           </property>
           <property name="verticalSpacing" >
            <number>6</number>
           </property>
           <item row="0" column="0" >
            <widget class="QRadioButton" name="posXAxis" >
             <property name="text" >
              <string>X Axis</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QRadioButton" name="negXAxis" >
             <property name="text" >
              <string>-X Axis</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QRadioButton" name="posZAxis" >
             <property name="text" >
              <string>Z Axis</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QRadioButton" name="negYAxis" >
             <property name="text" >
              <string>-Y Axis</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QRadioButton" name="posYAxis" >
             <property name="text" >
              <string>Y Axis</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QRadioButton" name="negZAxis" >
             <property name="text" >
              <string>-Z Axis</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QSlider" name="_distanceSlider" >
          <property name="minimum" >
           <number>0</number>
          </property>
          <property name="maximum" >
           <number>99</number>
          </property>
          <property name="sliderPosition" >
           <number>0</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_distanceSpinBox" >
          <property name="minimumSize" >
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum" >
           <double>10000000000.000000000000000</double>
          </property>
          <property name="singleStep" >
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="_advancedOptions" >
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>9</number>
         </property>
         <property name="topMargin" >
          <number>9</number>
         </property>
         <property name="rightMargin" >
          <number>9</number>
         </property>
         <property name="bottomMargin" >
          <number>9</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="_normalX" >
             <property name="minimum" >
              <double>-1.000000000000000</double>
             </property>
             <property name="maximum" >
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep" >
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="_normalY" >
             <property name="minimum" >
              <double>-1.000000000000000</double>
             </property>
             <property name="maximum" >
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep" >
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Z:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="_normalZ" >
             <property name="minimum" >
              <double>-1.000000000000000</double>
             </property>
             <property name="maximum" >
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep" >
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="_addButton" >
             <property name="text" >
              <string>Add Clip Plane</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="_removeButton" >
             <property name="text" >
              <string>Remove Clip Plane</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="_clipPlaneList" />
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
